app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,72 @@
  *= require foundation_and_overrides
  */
 
+ @import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+
+ @font-face {
+	font-family: 'icomoon';
+	src:url('../fonts/icomoon/icomoon.eot');
+	src:url('../fonts/icomoon/icomoon.eot?#iefix') format('embedded-opentype'),
+		url('../fonts/icomoon/icomoon.woff') format('woff'),
+		url('../fonts/icomoon/icomoon.ttf') format('truetype'),
+		url('../fonts/icomoon/icomoon.svg#icomoon') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
  /************** BASIC STYLING *****************************/
 
 #content {
-  background-color: #DADADA;
+  background-color: #FFF;
+  margin-top: 6rem;
+  margin-bottom: 6rem;
+}
+
+#background {
+  display: flex;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size, display, flex-flow, height, overflow